Functions/Initialization/xASL_init_InitializeMutex.m

```diff
@@ -64,16 +64,80 @@
 %% --------------------------------------------------------
 %% 2) Initialize mutex object
 x.mutex = xASL_GoNoGo(x.dir.LockDir);
+
+%% 3) Check if this module is locked by another process
+bLocked = 0;
+
 if ~x.mutex.Lock(x.settings.MUTEXID)
-    fprintf('Also, check that there is no filesystem permission issue\n');
 	fprintf(2,['ERROR in module_' x.ModuleName ': mutex is locked: %s in %s\n'], x.settings.MUTEXID, x.dir.LockDir);
-	fprintf('This means that this module is currently being parallel processed by another Matlab instance/worker\n');
+	fprintf('\n');
+    fprintf('This means that this module is currently being parallel processed by another Matlab instance/worker\n');
 	fprintf('If this is not the case, the locked folder needs to be removed before proceeding\n');
 	fprintf('Also, check that there is no filesystem permission issue\n');
     fprintf('Otherwise this error can be ignored\n');
-	return;
-end
+    fprintf('\n');
+
+    bLocked = 1;

```

Comment:
We sholdn't use this bLocked. 

The MUTEX has to work in exactly the same way as before. With the only exception that while creating a "lock", it will check if the entire subject is free of locks. 

But this should be done elsewhere.

Reply:
Now throughout bLocked -> x.mutex.bAnyModuleLocked

Reply:
Sounds good. Will check and test in detail.